src/test/java/io/xstefank/wildfly/bot/utils/JsonHandler.java

```diff
@@ -0,0 +1,83 @@
+package io.xstefank.wildfly.bot.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Class responsible for handling and changing properties of the JSON template file.
+ */
+public class JsonHandler {
+
+    private static JsonNode file;
+
+    /**
+     * Constructor for the JsonHandler class.
+     *
+     * @param fileName name of the JSON file located in 'src/test/resources/' folder. The name must contain the .json suffix.
```

Comment:
So you can't read json from file that doesn't have .json suffix?

Reply:
Good point, added the addition of .json suffix fallback